section3/exercises/arrays.js

```diff
@@ -96,21 +110,45 @@ YOU DO:
 Using the variables defined below, write a program that will tell a user if they
 will be able to call an Uber.
 
-The user can call an uber if they have 15% battery remaining, or more. In this case, it doesn't matter 
+The user can call an uber if they have 15% battery remaining, or more. In this case, it doesn't matter
 if the user has a charger at all, or what type.
 The can call an uber if they have a charger and it is a car charger.
 
+ 🤯 this is confusing because the statements in lines 112-114 directly contradict each other. It doesn't matter if they have a charger, but if they have a charger they can call a car? If the charger only works in the car, how can they use it to call the car with a dead phone? anyway...
```

Comment:
good feedback! It's not uncommon to get a problem description as a developer from someone that doesn't actually make sense if you think about it 😂